release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -882,49 +1023,34 @@ In the table below, features are marked with the following statuses:
 |GA
 |GA
 
-|External DNS Operator
-|TP
-|TP
-|TP
+|====
 
-|Alerting rules based on platform monitoring metrics
-|-
-|TP
-|TP
+[id="ocp-architecture-technology-preview-features"]
+=== Architecture technology preview features
 
-|AWS Load Balancer Operator
-|-
-|TP
-|TP
+.Architecture technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |OCP 4.10 |OCP 4.11 | OCP 4.12
 
-|Node Observability Operator
+|Hosted control planes for {product-title}
 |-
 |TP
 |TP
 
-|Java-based Operator
-|-
-|TP
-|TP
+|====
 
-|Hosted control planes for {product-title}
-|-
-|TP
-|TP
+[id="ocp-machine-management-technology-preview-features"]
+=== Machine management technology preview features
 
-|Managing machines with the Cluster API
-|-
-|TP
-|TP
+.Machine management technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |OCP 4.10 |OCP 4.11 | OCP 4.12
 
-|Linux Control Group version 2 (cgroup v2)
-|-
+|Managing machines with the Cluster API
 |-
 |TP
-
-|crun container runtime
-|-
-|-
 |TP
 
 |Cron job time zones
```

Comment:
I don't think this goes in this section (sorry, idk where it does go)

Reply:
I answered this in my review below :)